{
  "comments": [
    {
      "key": {
        "uuid": "7a881f9b_f7ff1985",
        "filename": "init/builtins.cpp",
        "patchSetId": 1
      },
      "lineNbr": 1101,
      "author": {
        "id": 1103203
      },
      "writtenOn": "2019-09-04T22:43:49Z",
      "side": 1,
      "message": "nit: consider also logging the message?",
      "range": {
        "startLine": 1101,
        "startChar": 0,
        "endLine": 1101,
        "endChar": 63
      },
      "revId": "95761d3d681beb34aeb71f6d59b792812568dabe",
      "serverId": "85c56323-6fa9-3386-8a01-6480fb634889",
      "unresolved": true
    },
    {
      "key": {
        "uuid": "cfee5209_3be45147",
        "filename": "init/builtins.cpp",
        "patchSetId": 1
      },
      "lineNbr": 1205,
      "author": {
        "id": 1003224
      },
      "writtenOn": "2019-09-04T22:48:50Z",
      "side": 1,
      "message": "i\u0027d rather have had a --reboot-on-failure option to exec, but this is a lot easier to implement...",
      "revId": "95761d3d681beb34aeb71f6d59b792812568dabe",
      "serverId": "85c56323-6fa9-3386-8a01-6480fb634889",
      "unresolved": true
    }
  ]
}